packages/twenty-server/src/modules/workflow/common/query-hooks/workflow-query-hook.exception.ts

```diff
@@ -0,0 +1,11 @@
+import { CustomException } from 'src/utils/custom-exception';
+
+export class WorkflowQueryHookException extends CustomException {
```

Comment:
Not sure if we want to call it QueryHookException? Maybe WorkflowVersionValidationException? Up to you!